src/main/resources/init_db.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE books (
+id BIGINT NOT NULL AUTO_INCREMENT,

```

Comment:
Primary Key couldn't be null, so such constraint is redundant
```suggestion
id BIGINT AUTO_INCREMENT,

```